products/couchbase-server.md

```diff
@@ -16,6 +16,11 @@ releaseDateColumn: true
 versionCommand: cat /opt/couchbase/VERSION.txt
 
 releases:
+-   releaseCycle: "7.1"
+    eol: no
```

Comment:
False

Reply:
Awwh, did the same mistake again :-( fixed it...